feature/experimental/bgp/otg_tests/base_bgp_session_parameters/metadata.textproto

```diff
@@ -0,0 +1,6 @@
+# proto-file: proto/metadata.proto
```

Comment:
This looks to have been created with a slightly dated version of addrundata. If you sync forward and rerun, it should output a path starting with github.com.

Reply:
Copied the file from the ate folder. For some reason I'm getting a failure while trying to generate it. btw...what is the correct command for that?